python/ray/ray_constants.py

```diff
@@ -159,7 +159,16 @@ def to_memory_units(memory_bytes, round_up):
 PROCESS_TYPE_GCS_SERVER = "gcs_server"
 
 WORKER_PROCESS_TYPE_IDLE_WORKER = "ray::IDLE"
-WORKER_PROCESS_TYPE_IO_WORKER = "ray::IOWorker"
+WORKER_PROCESS_TYPE_SPILL_WORKER_PREFIX = "ray::SpillWorker"
+WORKER_PROCESS_TYPE_RESTORE_WORKER_PREFIX = "ray::RestoreWorker"
+WORKER_PROCESS_TYPE_SPILL_WORKER_IDLE = (
```

Comment:
Could we call it IDLE_SpillWorker since process names are usually truncated to the first few characters in `top`?

Reply:
In this case, will it look like

IDLE: `ray::IDLE_SpillWorker`
spill: `ray::SPILL_SpillWorker`?

Reply:
Or do you completely want to remove ray::?

Reply:
Changed to 

IDLE: ray::IDLE_SpillWorker
spill: ray::SPILL_SpillWorker

IDLE: ray::IDLE_RestoreWorker
restore: ray::RESTORE_RestoreWorker